src/Component/Board/index.js

```diff
@@ -0,0 +1,20 @@
+import { connect } from 'react-redux'
+import { addCard } from '../../Redux/State/Board'
+import { compose } from 'ramda'
+import Board from './Board'
+
+// mapStateToProps :: State -> Props
+const mapStateToProps = state => ({
+  ...state.Board
```

Comment:
I prefer when i describe every prop, for example:
`cards: state.Board.cards,
isLoading: state.Board.isLoading,`
WDYT?